worker-api/src/main/java/com/hpe/caf/api/worker/WorkerCallback.java

```diff
@@ -24,20 +24,22 @@
 {
     /**
      * Used to send responses prior to the final response (when complete should be used instead).
-     *
-     * @param taskInformation a queue-specific reference for the incoming message that generated the response
+     *  @param taskInformation a queue-specific reference for the incoming message that generated the response
      * @param responseMessage the message to put on the queue (target specified by the {@code to} property)
+     * @param publishTaskDataAsObject boolean flag which decides whether the message will be published with
+     *                      taskData as object or byte[].
      */
-    void send(TaskInformation taskInformation, TaskMessage responseMessage);
+    void send(TaskInformation taskInformation, TaskMessage responseMessage, final boolean publishTaskDataAsObject);
```

Comment:
Interface parameters should not be marked `final` (unless it is a default method)
```suggestion
    void send(TaskInformation taskInformation, TaskMessage responseMessage, boolean publishTaskDataAsObject);
```